examples/tornjak/values-ingress.yaml

```diff
@@ -0,0 +1,64 @@
+global:
+  spire:
+    jwtIssuer: https://oidc-discovery.$APP_SUBDOMAIN
+  telemetry:
+    prometheus:
+      enabled: true
+
+spire-server:
+  tornjak:
+    enabled: true
+    ingress:
+      enabled: true
+      annotations:
+        route.openshift.io/termination: "edge"
+      hosts:
+        - host: spire-tornjak-backend.$APP_SUBDOMAIN
+          paths:
+            - path: /
+              pathType: Prefix
+
+  ingress:
+    enabled: true
+    annotations:
+      route.openshift.io/termination: "passthrough"
```

Comment:
In the other repo, we have some examples that are per ingress controller type..... maybe we should name things that way here since this looks to be specific to openshift?